admin/src/views/Agents.vue

```diff
@@ -9,55 +9,94 @@
         <el-form-item label="Agent name" prop='name'>
           <el-input v-model="newAgent.name"></el-input>
         </el-form-item>
-        <el-form-item label="NLU Conf">
-          <el-radio-group v-model="newAgent.configType" @change="selectConf">
-            <el-tooltip
-              class="item"
-              effect="dark"
-              content="Recommended when you have more than 1000 training data"
-              placement="top-start"
-            >
-              <el-radio label="supervised_embeddings">
-                Supervised Embeddings
-              </el-radio>
-            </el-tooltip>
+        <el-form-item label="Fallback">
             <el-tooltip
               class="item"
               effect="dark"
-              content="Recommended when you need a specific nlu pipeline"
+              content="percentage"
               placement="top-start"
             >
-              <el-radio label="specific_pipeline">
-                Specific Pipeline
-              </el-radio>
+              <el-input-number v-model="newAgent.fallback" :min="0" :max="100"></el-input-number>
             </el-tooltip>
-          </el-radio-group>
-        </el-form-item>
-        <el-form-item label="Language" :class="[isHideSpecificPipeline ? '' : 'displayNone']">
-          <el-select v-model="newAgent.language" placeholder="please select your zone">
-            <el-option label="English" value="en"></el-option>
-            <el-option label="French" value="fr"></el-option>
-          </el-select>
-        </el-form-item>
-        <el-form-item label="Specific pipeline" :class="[isHideSpecificPipeline ? 'displayNone' : '']">
-          <el-input type="textarea" :rows="10" v-model="newAgent.pipeline"></el-input>
-        </el-form-item>
-        <el-form-item label="Fallback">
-          <el-tooltip
-            class="item"
-            effect="dark"
-            content="percentage"
-            placement="top-start"
-          >
-            <el-input-number v-model="newAgent.fallback" :min="0" :max="100"></el-input-number>
-          </el-tooltip>
-        </el-form-item>
-        <el-form-item label="Training Data">
-          <el-input type="textarea" :rows="7" v-model="newAgent.trainingData"></el-input>
-        </el-form-item>
-        <el-form-item label="Responses">
-          <el-input type="textarea" :rows="7" v-model="newAgent.responses"></el-input>
+          </el-form-item>
+        <el-form-item label="Agent Creation">
+          <el-radio-group v-model="agentCreationOption"
+            @change="isWithImportAgentCreation = !isWithImportAgentCreation">
```

Comment:
maybe remove and use directly condition :
..... v-if="agentCreationOption === 'create_new_agent'"  .....